doc/source/index.rst

```diff
@@ -51,42 +53,34 @@ It's also possible you end up getting experienced with:
    For a more elaborate introduction to the tools and services that JupyterHub
    depends upon, see our :ref:`tools` page.
 
-
-
 .. _getting-to-zero:
 
-Setup a Kubernetes cluster
---------------------------
+Setup Infrastructure
+---------------------
 
 This section describes a Kubernetes cluster and outlines how to complete *Step Zero: your Kubernetes cluster* for
 different cloud providers and infrastructure.
+Since this project utilizes Helm as the package manager for Kubernetes deployments defined as charts,
+we will also cover how to set it up for your infrastructure.
 
 .. toctree::
-   :titlesonly:
-   :caption: Setup a Kubernetes cluster
+   :maxdepth: 2
 
-   create-k8s-cluster
+   infra/index
 
 .. _creating-your-jupyterhub:
 
 Setup JupyterHub
 ----------------
 
-This tutorial starts from *Step Zero: your Kubernetes cluster* and describes the
+This tutorial starts from *Step Zero: Your Kubernetes cluster* and describes the
 steps needed for you to create a complete initial JupyterHub deployment.
-This will use the JupyterHub Helm chart which provides sensible defaults for
-an initial deployment.
+Please ensure you have a working 
```

Comment:
This sentence seems incomplete

![image](https://user-images.githubusercontent.com/3837114/95729519-4d2a5f80-0c7d-11eb-88ff-04c442ee4de1.png)

Reply:
I'm growing old and blind! Will work on the whole thing this weekend! :runner: